code/modules/lighting/lighting_systems/europa_classic.dm

```diff
@@ -0,0 +1,72 @@
+// Still WIP for now...
+
+/datum/lighting_system/europa_classic
+	name = "Europa Classic"
+	desc = "A dynamic lighting system featuring soft shadows, special effects and bright colours. It aims to be reasonably bright and exude an atmosphere of a well-maintained station."
+	enabled = 1
+/*
```

Comment:
why even PR this one if it's all commented out?

Reply:
It's a wip that I had half an idea to bundle with Europa Lights' main PR before realising it was too much wokr and unatomic. I kinda wanted to get the draft files stored somewhere, but I can remove them if not needed. They have been unticked now.